introduction_to_amazon_algorithms/jumpstart-foundation-models/question_answering_retrieval_augmented_generation/question_answering_Cohere+langchain_jumpstart.ipynb

```diff
@@ -0,0 +1,1702 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    cohere_embedding_package = "cohere-multilingual-small-v1-1-e0210aae7f8135adaec4b199df604c4c"
</pre><p>Do we have other variants of emebdding models? I saw this is small size. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4409/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>